packages/x-license/src/utils/licenseModel.ts

```diff
@@ -10,9 +10,9 @@ export const LICENSING_MODELS = [
    */
   'annual',
   /**
-   * TODO 2025 remove, legacy name of annual.
+   * TODO 2026 remove, legacy name of annual.
    */
   'subscription',
 ] as const;
 
-export type LicensingModel = (typeof LICENSING_MODELS)[number];
+export type LicenseModel = (typeof LICENSE_MODELS)[number];
```

Comment:
This is public API strictly speaking
We can either keep exporting an `LicensingModel` next to `LicenseModel`, or don't care because nobody would actually use this variable

Reply:
We indeed use for this tool: https://tools-private.mui.com/prod/pages/x_licenseKey. But it's simple to handle on our side.

Reply:
But do we agree that it's a breaking change? Since it's exported from a package people install, and without an `unstable` or equivalent prefix.

Reply:
Technically, it's a breaking change, but we have never documented it and there is no practical use case for developers, so I think it's ok. 


We have https://github.com/mui/mui-x/issues/9346 about the root problem.